opencti-platform/opencti-front/src/components/dataGrid/dataTableUtils.tsx

```diff
@@ -906,25 +900,31 @@ const defaultColumns: DataTableProps['dataColumns'] = {
     label: 'File name',
     percentWidth: 12,
     isSortable: false,
-    render: (data, { column: { size } }) => {
+    render: (data) => {
       const file = (data.importFiles?.edges && data.importFiles.edges.length > 0)
         ? data.importFiles.edges[0]?.node
         : { name: 'N/A', metaData: { mimetype: 'N/A' }, size: 0 };
-      return (<Tooltip title={file?.name}><>{truncate(file?.name, size * MAGICAL_SIZE)}</>
-      </Tooltip>);
+      return (
+        <Tooltip title={file?.name}>
+          <Truncate>{file?.name}</Truncate>
+        </Tooltip>
+      );
     },
   },
   file_mime_type: {
     id: 'file_mime_type',
     label: 'Mime/Type',
     percentWidth: 8,
     isSortable: false,
-    render: (data, { column: { size } }) => {
+    render: (data) => {
       const file = (data.importFiles?.edges && data.importFiles.edges.length > 0)
         ? data.importFiles.edges[0]?.node
         : { name: 'N/A', metaData: { mimetype: 'N/A' }, size: 0 };
-      return (<Tooltip title={file?.metaData?.mimetype}><>{truncate(file?.metaData?.mimetype, size * MAGICAL_SIZE)}</>
-      </Tooltip>);
+      return (
```

Comment:
tooltip doesn't seem to be displayed in Artifact > Mime type column .... 
![image](https://github.com/user-attachments/assets/518ead32-b5de-462b-aad3-5cea460c4974)